src/main/java/io/rpg/model/object/GameObject.java

```diff
@@ -132,4 +142,8 @@ public enum Type {
       this.asString = asString;
     }
   }
+
+  public int getStrength() {
+    return strength;
+  }
```

Comment:
I suggest adding a comment here:

```suggestion
  // TODO: remove this getter when generic properties are implemented
  public int getStrength() {
    return strength;
  }
```